core/src/main/java/org/jruby/util/io/OpenFile.java

```diff
@@ -134,7 +136,7 @@ public void checkReopenSeek(ThreadContext context, Ruby runtime, long pos) {
 
     public EConv readconv;
     public EConv writeconv;
-    public IRubyObject writeconvAsciicompat;
+    public Encoding writeconvAsciicompat;
```

Comment:
Seems very unlikely to me this will break anything but it is public.  It is for 9.3 too so I guess we can break.  Just bringing it up.

Reply:
Yeah I played a little fast and loose with binary compat within OpenFile because nobody should be using it. Unfortunately it is in util.io package so these things have to be public to be accessible from RubyIO. Fingers crossed. 🤞